### ISSUE TITLE
composer: The Edit Filter box is way too small.

### ISSUE THREAD

**Issue by lloydtabb** [looker-open-source/malloy#600]
We need a better UI for the custom filters.  Often these are reasonably long strings.  We should make this box much larger and maybe allow wrapping.

**Comment by lloydtabb** [comment]
![image](https://user-images.githubusercontent.com/1093458/177050601-9b3a3459-b5b2-481a-bc93-73826f2b4c39.png)

**Comment by anikaks** [comment]
I agree! I'd also love to see autocomplete of some sort since I can never remember exactly what I called a given field, and for not all filter editing experiences to go fully custom like this; adding a filter is a much better experience than editing one currently

**Comment by anikaks** [comment]
Things filter editing should provide:
- Parity between editing & creating: the UI for creating filters is nice, we should show this while editing filters too
- wrapping & reasonable experience for long filter expressions in Custom option
- Auto-complete (you can't see the schema to figure out column names while filtering so this is pretty painful currently)

**Comment by christopherswenson** [comment]
@whscullin I can take this one, actually.
